### DISCUSSION TITLE
Empty state for a table in a filament

### DISCUSSION THREAD

**Discussion by SariMorgenshtern** [filamentphp/filament#14281] [Help]
### Package

Table builder

### Package Version

v3.0.0

### How can we help you?

I want in case I don't have rows in the table in Filament - I won't be shown an **empty stat**e.
that I can completely make the table disappear whether I have no data or if I have data.
When I have rows in the table I did this:
```
 public static function table(Table $table): Table
 {
 return $table->paginated(false);
 }
```
But if I have no rows, an empty state appears to me - while I want nothing to appear.
how can i do that
I didn't find a way in Filament's documentation...

**Comment by leandrocfe** [comment]
I think you can achieve this customizing this view.

Add this to your ListPage

```php
protected static string $view = 'custom-list-page';
```

```html
<!-- resources/views/custom-list-page.blade.php -->
<x-filament-panels::page>
    {{ $this->getAllTableRecordsCount() > 0 ? $this->table : '' }}
</x-filament-panels::page>
```

ref: https://filamentphp.com/docs/3.x/panels/resources/listing-records#custom-list-page-view

_To see everything that the default view contains, you can check the vendor/filament/filament/resources/views/resources/pages/list-records.blade.php file in your project._
